special-interest-groups/sig-tools/member-list.md

```diff
@@ -33,6 +33,11 @@ Co-Leads:
 
 ## Active Contributors
 
+* [lance6716](https://github.com/lance6716)
+
 ## Contributors
 
 * [AndrewDi](https://github.com/AndrewDi)
+* [glorv](https://github.com/glorv)
```

Comment:
what about `tsthght`, `djshow832`,`dixingxing0`, `freemindLi` from TiDB Binlog?

Reply:
`oscar-zyh` from ticdc

Reply:
`u5surf` from br

Reply:
these are the "1 PR contributors" i've mentioned in the description 😔